test/api/CypherTest.ts

```diff
@@ -16,15 +17,24 @@ chai.use(responseHelper);
 
 const FAKE_JSON: CypherJSON = { 'sensitiveData': ['content.secret1', 'content.secret2'], 'content': { 'secret1': 'Harry', 'secret2': 'Potter' } };
 const FAKE_WORD = 'Wingardium Leviosa';
-const FAKE_WORD_ENCRYPTED = '73d49673e8f70de6e16ba99bc0c273da:8b9cb41f960d2b50604921a68d7d0382472d';
+const FAKE_WORD_ENCRYPTED = '04480e7ab47eb5729faea20f0844dffa:81927d0bc660c4548b27cc0e3427d6874ea3';
```

Comment:
I think that unit test is not relevant anymore: its purpose was to detect a key change and the goal of your work is to allow key to change. Newly created tenant already have a random key. 
So I think you can just delete it. 
Unit tests is relevant to test that the key migration code work properly.

Reply:
I delete it then and create add in SettingTest.ts a condition if crypto setting is in DB